index.html

```diff
@@ -1,16 +1,105 @@
+                <li>About</li>
+            </ul>
+        
+        </nav>
+        <img class="cakeLogo" src="images/cake-logo.jpg" alt="logo" width="150px" height="150px">
+    
+        <!--Hello information and image carousel-->
+            <article class="intro">
+                <h1>Hello</h1>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                
+            </article>
 
-</html>
\ No newline at end of file
+            <section class="carousel">
+                    <!--making the image width 100% to make it responsive-->
+            <ul>
+                    
+                <li>
+                    <div class="carousel-container">
+                        <img src="images/cake.jpg" alt="Snow" style="width:100%">
+                    </div>
+                </li>
+                <li>
+                    <div class="carousel-container">
+                          <img src="images/cake.jpg" alt="Snow" style="width:100%">
+                    </div>
+                </li>
+                 <li>
+                    <div class="carousel-container">
+                        <img src="images/cake.jpg" alt="Snow" style="width:100%">
+                    </div>
+                </li>
+                <li>
+                    <div class="carousel-container">
+                        <img src="images/cake.jpg" alt="Snow" style="width:100%">
+                    </div>
+                </li>
+            </ul>
+        </section>
+
+            <!--4 row of images-->
+            <section class="three">
+                <div>
+                  <h2>Here is a range of our cakes</h2>
+                  <ul class="cake1">
+                    <li>
+                      <img src="image/cake.jpg" alt="cake" width="127" height="127">
+                      <h3>cupcakes</h3>
+                    </li>
+                    <li>
+                      <img src="images/cake.jpg" alt="ccake" width="127" height="127">
+                      <h3>cookie cake</h3>
+                    </li>
+                    <li>
+                      <img src="images/cake.jpg" alt="bcake" width="127" height="127">
+                      <h3>Brownie cake</h3>
+                    </li>
+                  </ul>
+                </div>
+              </section>
+
+            <!--Footer for sitemap etc...-->
+            <footer>
+                <nav>
+                    <ul class="footer-nav">
+                    <li><a href="#">Sitemap</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#">Delivery</a></li>
+                    </ul>
+                </nav>
+            </footer>
+        </main>
```

Comment:
The footer html element should appear after the `<main>` element - have a quick read to remind yourself of the [main](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) and [footer](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) html elements